eng/common/scripts/Create-APIReview.ps1

```diff
@@ -0,0 +1,87 @@
+[CmdletBinding()]
+Param (
+  [Parameter(Mandatory=$True)]
+  [string] $ArtifactPath,
+  [Parameter(Mandatory=$True)]
+  [string] $APIViewUri,
+  [Parameter(Mandatory=$True)]
+  [string] $APIKey,
+  [Parameter(Mandatory=$True)]
+  [string] $APILabel,
+  [string] $PackageName = ""
+)
+
+
+# Submit API review request and return status whether current revision is approved or pending or failed to create review
+function Submit-APIReview($packagename, $filePath, $uri, $apiKey, $apiLabel){
+    
+    $multipartContent = [System.Net.Http.MultipartFormDataContent]::new()
```

Comment:
Is calling all the .NET APIs here the best way to do this in PS? If this is the only way then so be it but this definitely looks a little unusual for PS code.

Reply:
Basic http request in powershell is not binding uploaded file to form fields in asp.net app ( IFormFile). This form has mixed form data content so I had to use .Net modules. Alternate choice was to use python script to keep it simple but I thought of going with PS script.